### PR TITLE
decode url and add hebrew massges

### PR DESCRIPTION
**Pull request by ashuvax**

### CODE REVIEW

**Comment by ashuvax** [comment]
The problem is that this development is not always good
For example when there is a space it does not look like a link
Maybe someone else will have a solution here
I thought maybe it could be done that any profit would be coded back by 20%

File: content_script.js [hunk 1]
```diff
@@ -79,6 +79,8 @@
   };
 
   const copyToClipboard = (url, selectedText) => {
+    // decode url
+    url = decodeURIComponent(url);
```
Comment (tomayac) [inline content_script.js:83]:
```suggestion
```

**Review by tomayac — APPROVED** [review]
LGTM, approval.

Thanks for contributing the Hebrew locale!

**Comment by tomayac** [comment]
Could you open a separate issue with the URL de-/encoding problem, please? Ideally with an example link?

**Comment by ashuvax** [comment]
I could not understand you dear what is the problem?

**Comment by tomayac** [comment]
I have not merged the part https://github.com/GoogleChromeLabs/link-to-text-fragment/pull/52/commits/6d02fca36add74a1f009900e267787050ab5f851#diff-487c8987f248f38f59ac47f7076cdb888ca36d7e38275c36269a1653d3cdaa1b. Please describe with example links what problem this was solving. Thanks!

**Comment by ashuvax** [comment]
Now I think it might be possible to add:
`url = decodeURIComponent(url);
url = url.replaceAll(" ", "%20");`

**Comment by tomayac** [comment]
Oh, I see. What you mean is called percent-encoding and it is used here to represent Hebrew characters in URLs. You can play with this encoding at https://meyerweb.com/eric/tools/dencoder/. Paste in some Hebrew words and press _encode_.

**Comment by ashuvax** [comment]
What I recommend is that instead of doing it through the site you specified it be automated in your plugin

**Comment by tomayac** [comment]
This already is this case. See `https://example.com/#:~:text=example%20domain,-This` for example, which is the link that is generated when you select the heading on [example.com](https://example.com/):

Screen Shot 2022-01-31 at 15 41 13

**Comment by ashuvax** [comment]
Do not understand you Do you understand Hebrew?
I just use with Google Translate so it looks like this
I ask why you do not want to enter the code I brought you?
It will help that links that have Hebrew words in it will make a link beautiful and not as long as I showed you

**Pull request merged**